### ISSUE TITLE
MULEbots either don't have sided sprites or don't use them

### ISSUE THREAD

**Issue by TheVekter** [tgstation/tgstation#76820]
If they move any direction other than to the east their sprite won't respond properly, so it just looks like they're hovering.

**Comment by Jacquerel** [comment]
This is true but it is also true that _crates_ don't have directional sprites.
Because MULEbots need to carry crates as overlays, they need to always look correct with a crate on top of them.
This makes directional sprites either a challenge or actively impossible.

**Comment by TheVekter** [comment]
![image](https://github.com/tgstation/tgstation/assets/7019927/69142a4d-f369-4bf9-ae99-1a157999e994)

I guess we need crate directionals too unless we're just okay with floating mulebots which... I guess, but it's always going to bother me.

**Comment by ArcaneMusic** [comment]
There is, if you *really* wanted to have a specific fix for this, a unique middle ground which is actually based in real life:

<img width="2000" height="1999" alt="Image" src="https://github.com/user-attachments/assets/299b8e51-910c-4a79-9d4d-41f5989f7e7a" />
Mechanum wheels for robotics use these angled wheels so that they can strafe without needing to turn and can maintain their orientation in all directions. If we tweaked the sprites for mulebots to use these, it would stand to reason why they're always facing the long way.
